src/um/winbase.rs

```diff
@@ -1960,20 +1960,48 @@ extern "system" {
     // pub fn BackupEventLogA();
     // pub fn BackupEventLogW();
     // pub fn CloseEventLog();
-    // pub fn DeregisterEventSource();
+    pub fn DeregisterEventSource(
+        hEventLog: HANDLE,
+    ) -> BOOL;
     // pub fn NotifyChangeEventLog();
     // pub fn GetNumberOfEventLogRecords();
     // pub fn GetOldestEventLogRecord();
     // pub fn OpenEventLogA();
     // pub fn OpenEventLogW();
-    // pub fn RegisterEventSourceA();
-    // pub fn RegisterEventSourceW();
+    pub fn RegisterEventSourceA(
+        lpUNCServerName: LPCSTR,
+        lpSourceName: LPCSTR,
+    ) -> HANDLE;
+    pub fn RegisterEventSourceW(
+        lpUNCServerName: LPCWSTR,
+        lpSourceName: LPCWSTR,
+    ) -> HANDLE;
     // pub fn OpenBackupEventLogA();
     // pub fn OpenBackupEventLogW();
     // pub fn ReadEventLogA();
     // pub fn ReadEventLogW();
-    // pub fn ReportEventA();
-    // pub fn ReportEventW();
+    pub fn ReportEventA(
+        hEventLog: HANDLE,
+        wType: WORD,
+        wCategory: WORD,
+        dwEventID: DWORD,
+        lpUserSid: PSID,
+        wNumStrings: WORD,
+        dwDataSize: DWORD,
+        lpStrings: *const LPCSTR,
```

Comment:
This should be `*mut LPCSTR` according to the SDK

Reply:
Fixed.